### PR TITLE
adds mushrooms and chocolate to the produce orders

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

Chocolate is used in some high paying bartending drinks, chanterelle is used in chawanmushi but is TECHNICALLY obtainable from waiting for botany to grow mushrooms. basically, they should have been in the produce console to begin with!

## Why It's Good For The Game

Without this you have to constantly send away high payers. Hacking machine for fernet is fine as it's an investment, this is just CBT (crikey, bad tomfoolery)

## Changelog
:cl:
qol: chanterelle mushrooms and chocolate are now in the produce console
/:cl:

### CODE REVIEW

**Comment by Qustinnus** [comment]
i think this is QoL not balance as it was an oversight if anything. if a maint disagrees they can change it.

File: code/game/machinery/computer/chef_orders/order_datum.dm [hunk 1]
```diff
@@ -99,6 +99,16 @@
...
 
+/datum/orderable_item/chanterelle
+	name = "Chanterelle"
+	category_index = CATEGORY_FRUITS_VEGGIES
+	item_instance = /obj/item/food/grown/mushroom/chanterelle
+
+/datum/orderable_item/cocoa
+	name = "Cocoa"
+	category_index = CATEGORY_FRUITS_VEGGIES
+	item_instance = /obj/item/food/grown/cocoapod
```
Comment (Tlaltecuhtli) [inline code/game/machinery/computer/chef_orders/order_datum.dm:110]: this is not chocolate

**Comment by Time-Green** [comment]
I'm not sure that everything should be orderable on the produce console, some bots just shouldn't be servable without working with botany. And if the intent is to make everything that the bots can ask for orderable, you might also wanna add chili to the produce console

**Comment by necromanceranne** [comment]
> I'm not sure that everything should be orderable on the produce console, some bots just shouldn't be servable without working with botany. And if the intent is to make everything that the bots can ask for orderable, you might also wanna add chili to the produce console

it's not really necessarily efficient to order, and especially so if you're express ordering, so it largely balances out. You want to order goods ahead of orders, and in bulk as to not waste the console cooldown (basically ordering for a few of the same recipe). The console is still less effective than free goods out of botany for both time and value, and you'll never be making money if you're using ordered goods. But there are so many core plants for botany to provide that they probably simply won't, not out of malice, but because that's just a lot of plants to handle at once while doing their own stuff.

There is also a slight issue that botany _cannot_ make chanterelle because it randomly grows in untended trays. To me that seems silly, and the mycelium should be in the vendor. Even if it's premium.

**Comment by Tlaltecuhtli** [comment]
the yellow mushrooms are in the vendor what mushrooms are you smoking

**Comment by necromanceranne** [comment]
nyoops.

**Comment by tralezab** [comment]
Guys you're talking about restricting the ordering of a plant that will grow for free if nobody is in botany there's no big bank break going on here

**Pull request merged**